pkg/lib/saml/samlprotocol/const.go

```diff
@@ -29,3 +29,15 @@ const (
 	// https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf 3.2.2
 	SAMLVersion2 string = "2.0"
 )
+
+const (
+	SAMLIssertFormatEntity = "urn:oasis:names:tc:SAML:2.0:nameid-format:entity"
+)
+
+const (
+	SAMLAttrnameFormatBasic = "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
+)
+
+const (
+	SAMLAttrTypeString = "xs:string"
```

Comment:
`xs:` seems to be a XML namespace, which depends on how the namespace is named at the root of XML document. Does crewsaml exposes this constant so that we do not need to repeat it here? If not, we better document this hack, and saying upgrading crewsaml may break our code.

Reply:
`xs:string` is used to represent the type string in xml schema. Which I don't think is related to the crewsaml implementation.

However I don't know which spec exactly defined this, I only know it is commonly used.
The only related spec I found is https://www.w3.org/TR/xmlschema-2/

Reply:
Copy https://github.com/crewjam/saml/blob/main/schema.go#L502